dev/changelog/pullrequest.go

```diff
@@ -67,6 +78,7 @@ func init() {
 	prFlags.StringVarP(&prOpts.HeadBranch, "head", "H", "main", "the head branch for pull requests")
 	prFlags.StringVarP(&prOpts.BaseBranch, "base", "b", "main", "the base branch for pull requests")
 	prFlags.StringVarP(&prOpts.Title, "title", "T", "[changelog] updated changelog", "the title of the PR")
-	prFlags.StringVarP(&prOpts.Body, "body", "B", "Updated the changelog from recent PR descriptions\n\n```release-note\nNONE\n```", "the body of the PR")
+	prFlags.StringVarP(&prOpts.Body, "body", "B", "Updated the changelog from recent PR descriptions\n\n```release-note\nNONE\n```\n- [x] /werft no-preview\n- [x] /werft no-test", "the body of the PR")
```

Comment:
why is it adding the `no-preview` flag by default?

Reply:
I want to avoid unnecessary builds/deploys. It's just a changelog change. It should be auto-merged. We need a PR because of our branch protection only and the labels / comment because of prow.

Reply:
got it, it's limited to `PR_BRANCH=$(date '+rq/changelog_%Y%m%d%H%M')`